sdk/build.gradle

```diff
@@ -25,6 +25,7 @@ subprojects {
             testInstrumentationRunner "androidx.test.runner.AndroidJUnitRunner"
             versionCode versionFor(project, "version.klaviyo.versionCode") as Integer
             versionName versionFor(project, "version.klaviyo.versionName") as String
+            consumerProguardFiles("consumer-rules.pro")
```

Comment:
and each module with consumer-rules would need this line?

Reply:
I believe we only need to do this for the root level build.gradle, it does need to be under the `defaultConfig`

Reply:
Yeah I think that's right since this section applies to all the subprojects